tools/pydantic_codegen/automations/automations.toml

```diff
@@ -0,0 +1,39 @@
+# NOTE: Paths are relative to the root of the repo
+
+[tool.ariadne-codegen]
+schema_path = "core/api/graphql/schemas/schema-latest.graphql"
+queries_path = "tools/pydantic_codegen/automations/queries.graphql"
+
+files_to_include = ["tools/pydantic_codegen/base.py"]
+
+modules_to_drop = [
+    "async_base_client",
+    "base_client",
+    # We'll swap in a module with our own custom base class
+    "base_model",
+    "client",
+    "exceptions",
+]
+
+
+target_package_path = "wandb/sdk/automations"
+target_package_name = "_generated"
+
+include_all_inputs = false # Only generate input types needed for queries
+include_all_enums = false  # Only generate enums needed for queries
+
+plugins = [
+    "ariadne_codegen.contrib.extract_operations.ExtractOperationsPlugin",
+    "pydantic_codegen.plugin.ReplaceBaseModel",
+    "pydantic_codegen.plugin.FixFragmentOrder",
+    "pydantic_codegen.plugin.WandbCodegenPlugin",
```

Comment:
**note:** these are custom plugin extensions -- not supported by `ariadne-codegen` out of the box, but implemented following their [plugin docs](https://github.com/mirumee/ariadne-codegen/blob/main/PLUGINS.md)